models/stock_picking.py

```diff
@@ -715,3 +720,14 @@ def open_stock_picking_form_view(self):
             'res_id': self.id,
             'context': dict(self.env.context),
         }
+
+    @api.multi
+    def _refine_picking(self, text=None):
+        """
+        By default refining the picking is cancel
+        """
+        for pick in self:
+            if text:
+                pick.message_post(body=_(text))
+            pick.action_cancel()
+        return
```

Comment:
We use implicit `return`s elsewhere.

Reply:
https://www.python.org/dev/peps/pep-0020/

"explicit is better than implicit" is like the second tenant of the zen of python :-)